devai-cli/src/devai/utils.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
lets delete this file since there's a duplicate under:

devai-cli/src/devai/commands/utils.py